sys-process/below/below-0.2.0.ebuild

```diff
@@ -0,0 +1,220 @@
+futures-util-0.3.15
+getrandom-0.1.16
+getrandom-0.2.3
+half-1.7.1
+heck-0.3.3
+hermit-abi-0.1.18
+hostname-0.3.1
+humantime-1.3.0
+ident_case-1.0.1
+instant-0.1.9
+iovec-0.1.4
+itertools-0.8.2
+itoa-0.4.7
+jobserver-0.1.22
+kernel32-sys-0.2.2
+lazy_static-1.4.0
+libbpf-cargo-0.7.1
+libbpf-rs-0.10.0
+libbpf-sys-0.3.0-2
+libc-0.2.96
+lock_api-0.4.4
+log-0.4.14
+maplit-1.0.2
+match_cfg-0.1.0
+memchr-2.4.0
+memmap-0.7.0
+mio-0.6.23
+mio-0.7.11
+mio-named-pipes-0.1.7
+mio-uds-0.6.8
+miow-0.2.2
+miow-0.3.7
+net2-0.2.37
+nix-0.17.0
+nix-0.19.1
+ntapi-0.3.6
+num-0.3.1
+num-complex-0.3.1
+num-integer-0.1.44
+num-iter-0.1.42
+num-rational-0.3.2
+num-traits-0.2.14
+num_cpus-1.13.0
+num_enum-0.5.1
+num_enum_derive-0.5.1
+numtoa-0.1.0
+once_cell-1.7.2
+openat-0.1.21
+os_info-2.0.1
+owning_ref-0.4.1
+parking_lot-0.11.1
+parking_lot_core-0.8.3
+paste-1.0.5
+pest-2.1.3
+pin-project-lite-0.1.12
+pin-project-lite-0.2.6
+pin-utils-0.1.0
+pkg-config-0.3.19
+plain-0.2.3
+ppv-lite86-0.2.10
+proc-macro-crate-0.1.5
+proc-macro-error-1.0.4
+proc-macro-error-attr-1.0.4
+proc-macro-hack-0.5.19
+proc-macro-nested-0.1.7
+proc-macro2-1.0.27
+quick-error-1.2.3
+quote-1.0.9
+rand-0.4.6
+rand-0.7.3
+rand-0.8.3
+rand_chacha-0.2.2
+rand_chacha-0.3.1
+rand_core-0.3.1
+rand_core-0.4.2
+rand_core-0.5.1
+rand_core-0.6.2
+rand_hc-0.2.0
+rand_hc-0.3.0
+rand_pcg-0.2.1
+rdrand-0.4.0
+redox_syscall-0.2.8
+redox_termios-0.1.2
+redox_users-0.4.0
+regex-1.5.4
+regex-syntax-0.6.25
+remove_dir_all-0.5.3
+rustversion-1.0.5
+ryu-1.0.5
+same-file-1.0.6
+scopeguard-1.1.0
+scroll-0.10.2
+scroll_derive-0.10.5
+semver-0.11.0
+semver-0.9.0
+semver-parser-0.10.2
+semver-parser-0.7.0
+serde-1.0.118
+serde_cbor-0.11.1
+serde_derive-1.0.118
+serde_json-1.0.64
+signal-hook-0.1.17
+signal-hook-0.3.9
+signal-hook-registry-1.4.0
+slab-0.4.3
+slog-2.7.0
+slog-async-2.6.0
+slog-term-2.8.0
+smallvec-1.6.1
+stable_deref_trait-1.2.0
+static_assertions-1.1.0
+strsim-0.8.0
+strsim-0.9.3
+structopt-0.3.21
+structopt-derive-0.4.14
+strum_macros-0.18.0
+syn-1.0.73
+take_mut-0.2.2
+tempdir-0.3.7
+tempfile-3.2.0
+term-0.7.0
+termion-1.5.6
+textwrap-0.11.0
+thiserror-1.0.25
+thiserror-impl-1.0.25
+thread_local-1.1.3
+threadpool-1.8.1
+time-0.1.43
+tokio-0.2.25
+tokio-macros-0.2.6
+toml-0.5.7
+ucd-trie-0.1.3
+unicode-segmentation-1.7.1
+unicode-width-0.1.8
+unicode-xid-0.2.2
+users-0.9.1
+vec_map-0.8.2
+version_check-0.9.3
+void-1.0.2
+vsprintf-2.0.0
+walkdir-2.3.2
+wasi-0.10.2+wasi-snapshot-preview1
+wasi-0.9.0+wasi-snapshot-preview1
+wasmer_enumset-1.0.1
+wasmer_enumset_derive-0.5.0
+winapi-0.2.8
+winapi-0.3.9
+winapi-build-0.1.1
+winapi-i686-pc-windows-gnu-0.4.0
+winapi-util-0.1.5
+winapi-x86_64-pc-windows-gnu-0.4.0
+ws2_32-sys-0.2.1
+xi-unicode-0.3.0
+zstd-0.7.0+zstd.1.4.9
+zstd-safe-3.1.0+zstd.1.4.9
+zstd-sys-1.5.0+zstd.1.4.9
+"
+
+inherit cargo
+
+DESCRIPTION="An interactive tool to view and record historical system data"
+HOMEPAGE="https://github.com/facebookincubator/below"
+SRC_URI="$(cargo_crate_uris ${CRATES})"
```

Comment:
That SRC_URI is not complete. You are aware that the ebuild can't work this way? Please test your stuff before creating a pull request!